### PR TITLE
Changelog for v4.0.0

### PR DESCRIPTION
**Pull request by narqo**
relates to #1310

### CODE REVIEW

**Comment by narqo** [comment]
@tadatuta could you help with the translation?

**Comment by tadatuta** [comment]
sure, will do it tonight

**Comment by tadatuta** [comment]
@narqo added few commits to current branch

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
 # Changelog
 
+## 4.0.0
+
+### Изменения, ломающие обратную совместимость
```
Comment (narqo) [inline CHANGELOG.md]: This is just a stub for further translation, isn't it?

Comment (tadatuta) [reply CHANGELOG.md]: oh, it's just commit of not saved changes. fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
 
+## 4.0.0
+
+### Breaking changes
+
+- Changes in the `i-bem` block. See [MIGRATION.en.md](MIGRATION.en.md#changes-in-the-i-bem-block).
+- Changes in the `querystring`. See [MIGRATION.en.md](MIGRATION.ru.md#changes-in-the-querystring-block).
+- The optional parameter `onlyGet` was removed from the `identify` module ([#1028](https://github.com/bem/bem-core/issues/1028)).
+- All static methods were removed from the `events` module ([#1024](https://github.com/bem/bem-core/issues/1024)).
+- The `result` field in `Event` class of `events` module was removed  ([#1023](https://github.com/bem/bem-core/issues/1023)).
```
Comment (narqo) [inline CHANGELOG.md:11]: Guess, it's "of _the_ `events` module".

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+- The `css` element of the `page` block does not support auto insertion of conditional comments for IE ([#379](https://github.com/bem/bem-core/issues/379)).
```
Comment (narqo) [inline CHANGELOG.md:12]: I'd preferred to start the phrase with "Since now on" or similar. What do you think?

**Pull request merged**